core/Macros.carp

```diff
@@ -284,6 +284,13 @@
 (defmacro ignore [form]
   (list 'let (array '_ form) (list)))
 
+;; Also expose this as 'void' for those familiar with C?
+(doc unit "Convenience macro for specifing a return type of unit '()' for
```

Comment:
Nitpick: could we limit this to 80 chars per line?

Reply:
Good catch! Will do.

Reply:
Ah, actually, I'm missing something here--I'm seeing these come in at col 73 and 74

Reply:
Huh, somehow I’m now also seeing these as split lines. I dunno what happened with my brain there.

Reply:
No sweat!